api/src/main/java/org/pmiops/workbench/db/dao/WorkspaceDao.java

```diff
@@ -152,4 +161,35 @@ default WorkspaceActiveStatus getActiveStatusEnum() {
       return DbStorageEnums.workspaceActiveStatusFromStorage(getActiveStatus());
     }
   }
+
+  interface WorkspaceCostView {
+    Long getWorkspaceId();
+
+    String getGoogleProject();
+
+    Long getCreatorId();
+
+    Double getFreeTierCost();
+
+    Timestamp getFreeTierLastUpdated();
+
+    Timestamp getWorkspaceLastUpdated();
+
+    Short getActiveStatus();
+  }
+
+  @Query(
+      "SELECT w.workspaceId AS workspaceId, "
+          + "w.googleProject AS googleProject, "
+          + "w.creator.id AS creatorId, "
+          + "f.cost AS freeTierCost, "
+          + "f.lastUpdateTime AS freeTierLastUpdated, "
+          + "w.lastModifiedTime AS workspaceLastUpdated, "
+          + "w.activeStatus AS activeStatus "
+          + "FROM DbWorkspace w "
+          + "LEFT JOIN DbWorkspaceFreeTierUsage f ON w.workspaceId = f.workspace.id "
+          // Billing migration status 1 = NEW
```

Comment:
I think Billing migration status is a deprecated field. Do we have any row in DB that not equal to 1?

Reply:
In test db
select count(1) from workspace where billing_migration_status <> 1;
+----------+
| count(1) |
+----------+
|     6320 |
+----------+

Reply:
I believe we should keep it in the query

Reply:
Having w.billingMigrationStatus = 1 or not having will return the same query result, why we should keep it in the query?  
Having query here will make future deprecation harder.